common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -683,14 +671,17 @@ def _render_student_view_for_items(self, context, display_items, fragment, view=
                 content = rendered_item.content
             else:
                 content = ''
+
+            contains_content_type_gated_content = self._check_children_for_content_type_gating_paywall(item) is not None
             iteminfo = {
                 'content': content,
                 'page_title': getattr(item, 'tooltip_title', ''),
                 'type': item_type,
                 'id': text_type(usage_id),
                 'bookmarked': is_bookmarked,
                 'path': " > ".join(display_names + [item.display_name_with_default]),
-                'graded': item.graded
+                'graded': item.graded,
+                'contains_content_type_gated_content': contains_content_type_gated_content
```

Comment:
Nit: end this with a comma so if we ever add anything more, it results in a smaller diff.